gateway/src/integration-test/resources/http/grafana/query/extract-algo/testMissingAdd/testInvalidPointsSize/testQuery/expectedResponse.json

```diff
@@ -0,0 +1,9 @@
+[
+  {
+    "target": "openSpaceSensors.Temperature222",
+    "datapoints": [
+      [1477917224866, 8.0],
+      [1477895624866, 43.0]
+    ]
+  }
+]
```

Comment:
The point   [1477916224866, 4.0] of openSpaceSensors.Temperature111 should be returned as well isn't it ? Maybe it is not in the range of the query ? Anyway you should verify it is indeed injected.